standalone/node/src/chain_spec.rs

```diff
@@ -380,7 +370,17 @@ fn testnet_genesis(
 			authority_ids: initial_authorities.iter().map(|(x, ..)| x.clone()).collect::<_>(),
 		},
 		dkg_proposals: DKGProposalsConfig { initial_chain_ids, initial_r_ids, initial_proposers },
-		bridge_registry: Default::default(),
+		bridge_registry: pallet_bridge_registry::pallet::GenesisConfig {
+			bridges: bounded_vec![BridgeMetadata {
+				resource_ids: bounded_vec![RESOURCE_ID_HERMES_ATHENA, RESOURCE_ID_ATHENA_HERMES],
+				info: BridgeInfo {
+					additional: Default::default(),
+					display: SerdeData::from_str("hermes-athena")
+						.unwrap()
+				}
+			}],
+			..Default::default()
+		},
```

Comment:
Here are the bridge parameters being passed in. This works and shows up as expected in polkadot web ui. However the test case in https://github.com/webb-tools/relayer/pull/355 is failing now with `Metadata(IncompatibleStorageMetadata("BridgeRegistry", "Bridges"))`, I think its necessary to update the dependency in webb-rs crate to fix it.

Reply:
Yup, this needs an Update to webb-rs, can you generate the new metadata locally and use the local version of webb-rs to test that?

Reply:
I did that according to instructions, files `src/substrate/dkg_runtime.rs` and `src/substrate/protocol_substrate_runtime.rs` got updated. I changed the relayer webb dependency to use the local repo path, however its still giving the same error. Not sure what Im missing.